### DISCUSSION TITLE
about SCIP version 8.0.3

### DISCUSSION THREAD

**Discussion by motokiS2** [google/or-tools#3650] [Linear Solver questions] [answered]
I would like to use SCIP version 8.0.3, although the November 2022 release notes indicate that SCIP version 8.0.1 is now supported.
When will SCIP version 8.0.3 be supported?

**Comment by Mizux** [comment, marked as answer]
it is supported on `main` branch
https://github.com/google/or-tools/blob/73e4f754456f512edc450286dbe6929f4abeb5d8/cmake/dependencies/CMakeLists.txt#L228-L233

note: we would like to generate a new release ASAP but no date...
Still few last feature to integrate (pdlp python support, bazel java test support etc...)

**Comment by motokiS2** [comment]
thank you very much
